config.json

```diff
@@ -107,9 +107,15 @@
       "topics": []
     },
     {
-      "difficulty": 1,
+      "difficulty": 2,
```

Comment:
> I think it would be a more difficult task to rate all the exercises in one fell swoop and then add topics as we go. Would someone need to have completed all the exercises to do this? If they're reviewing the exercise to gauge difficulty, is it not worth making a note of topics simultaneously? Or would the initial difficulty ratings be a quick and simple best guess effort, to be adjusted over time?

My suspicion is that if we think the track is already well-ordered, we can take it slow.

My experience in another track: In Haskell, it was slightly urgent to get the track ordered quickly since some problems were in wildly wrong places, so I did everything in one go by roughly guessing how I'd approach the problem and estimating the difficulty of that approach, resulting in https://github.com/exercism/xhaskell/pull/402 . I had not completed every exercise personally, but I was at least familiar with the tests being given in every single exercise, since I had reviewed PRs updating every single one of them. I also cross-referenced with F# as that track has implemented almost every exercise and assigned difficulties to each one. You're right that these guesses could have been wrong, but it was better than the alternative.

That approach may not be necessary for this track. I had previous work checking ordering in #279, but this only checked stub files vs not. I haven't checked other things about track ordering.

At a quick glance, I would say react is probably too early, and variable-length-quantity probably doesn't belong at the very end.

> My intention from here is to add topics to exercises in the current order they are in, I only added grains out of step as I completed it yesterday and wanted to tackle it while fresh in my mind.

Ah, that explains it. Thanks. I was wondering why you suddenly jumped to grains...